docs/history/4_x/version-4.6.0.md

```diff
@@ -0,0 +1,77 @@
+# Release 4.6.0
+
+Name: <span style="color: salmon"><span class="glyphicon glyphicon-bullhorn"></span> "Ogre salmon bullhorn"</span>
+Release Date: August 31, 2022
+
+- [Download the Releases](https://download.rundeck.com/)
+- [Sign up for Release Notes](https://www.rundeck.com/release-notes-signup)
+- [Upgrade instructions](/upgrading/)
+
+## Overview
+
+Check out the new features and enhancements for PagerDuty Process Automation (formerly Rundeck Enterprise) and PagerDuty Runbook Automation and Rundeck Community included in this release. This release includes a new plugin for AWS Athena that job definitions to include SQL queries against Amazon S3, an enhancement of the Amazon ECS node source plugin to handle multiple clusters within the same region, and a number of security and compliance enhancements and bug fixes.
```

Comment:
This release includes a new plugin for AWS Athena that **enables** job definitions to include SQL queries against Amazon S3